constitution.tex

```diff
@@ -794,6 +794,7 @@
 	\item The candidates will be notified of their nomination.
 		Each candidate is given a minimum of twenty-four hour period to accept or decline the nomination.
 		A list of all nominees who have accepted their nominations will be posted shortly thereafter.
+	\item The date and time of the candidate(s)'s speeches will be announced by a member of the Executive Board at least five (5) days prior to the event. This will be treated as a House Meeting, with the expectation that all Active Members will attend and attendance will be taken. If the speeches take place at a traditional House Meeting, only one attendance will be taken.
```

Comment:
I don't like `this will be treated as a House Meeting`. If you want to define this as a requirement, please actually do so, akin to the other requirements. You can keep a section here defining what qualifies as an election for required attendance.